tests/workload/params_test.py

```diff
@@ -2862,20 +2862,20 @@ def test_params_default(self):
             Context.QUERY,
             self.data_set_dir
         )
-        neighbors_data_set_path = create_data_set(
+        create_data_set(
```

Comment:
When is this being used?

Reply:
Good point, In this case we are creating neighbor's dataset as part of query's dataset. In first step, you see that query dataset is created and it returns the dataset path. In step two, we pass same file and in this case, we don't create new file, instead create new dataset with key as neighbors in same file.